Mikado/_transcripts/transcript_base.py

```diff
@@ -1294,6 +1295,8 @@ def load_dict(self, state, trust_orf=False, accept_undefined_multi=False):
                 setattr(self, metric, state[metric])
             except KeyError:
                 raise KeyError((self.id, metric))
+        self._calculate_cds_tree()
+        self._calculate_segment_tree()
```

Comment:
Does this not make it slower for multiprocessing?

Reply:
It might when sending the transcript back to the main process, you are right. I am restoring the old behaviour now I fixed the calls to create the CDS tree.